src/executeMainThread.ts

```
@@ -26,6 +26,12 @@ import Axios from 'axios';
 import { TarService } from './services/tarService.js';
 
 export const executeMainThread = async () => {
+	const slicedArgv = hideBin(process.argv);
+
+	if (slicedArgv.length === 0) {
+		return;
```

Maybe throw an error? or show "help"?